Package.swift

```diff
@@ -12,9 +12,20 @@
 
 
 import PackageDescription
+import class Foundation.ProcessInfo
+
+let macOSPlatform: SupportedPlatform
+if let deploymentTarget = ProcessInfo.processInfo.environment["SWIFT_TOOLCHAIN_MACOS_DEPLOYMENT_TARGET"] {
```

Comment:
@abertelrud are we also changing SwiftPM to use this instead?

Reply:
Yes, that's the plan, although this needs to be coordinated with @shahmishal.  But I can put up a SwiftPM and SwiftDriver change to favour `SWIFT_TOOLCHAIN_MACOS_DEPLOYMENT_TARGET` and for now falling back on `SWIFTPM_MACOS_DEPLOYMENT_TARGET` until the new one is set in toolchain builds.